core/src/main/scala/za/co/absa/spline/harvester/builder/write/WriteNodeBuilder.scala

```diff
@@ -41,12 +44,14 @@ class WriteNodeBuilder
       id = operationId,
       name = command.name.asOption,
       childIds = childIds,
-      params = Map(command.params.toSeq: _*).asOption,
+      params = ioParamsConverter.convert(command.params).asOption, // TODO why was not used the original map in the previous code?
```

Comment:
That's fine, we don't need a `Map` wrapper here.

It's a leftover from the earlier versions where there was no `extra`, and `params` were used for carrying Spline specific things as well, hence extra keys were added to the map via `+` or `++`.
The problem used to be that since Spark maps are often case-insensitive, the resultant map could also be case-insensitive. That fact was very hard to notice and created an unwanted effect on the serialized DTOs where all added keys could appear unexpectedly lowercased.

Since we don't explicitly add any keys to `params` anymore, we can safely get rid of the wrapper.